### ISSUE TITLE
Allow multi-connection deploy target

### ISSUE THREAD

**Issue by coreypmurphy** [AutoCloud/ember-cli-deploy-simply-ssh#10]
Would love the ability to deploy to a "deployTarget" environment with multiple servers in same block.  For example, I have 2 production servers that need the same deployment however I would have to run two separate deploy operations given the current implementation.  It would be awesome to see the following where "connections" is an array of connection objects that deploys to each given one "ember deploy production" statement:

```
ENV['simply-ssh'] = {
      connections: [{
        host: 'SERVER_1',
        port: 22,
        username: 'USERNAME',
        privateKey: process.env.SSH_KEY
      },
      {
        host: 'SERVER_2',
        port: 22,
        username: 'USERNAME',
        privateKey: process.env.SSH_KEY
      }],
      dir: '/var/www/project',
      keep: 5
    }
```

**Comment by ryedeer** [comment]
Good idea! We'll try and implement it when time permits. However, if you'd like to implement it and make a PR, we'd be happy to merge it.

**Comment by NetForce1** [comment]
@ryedeer We need this as well, and I managed to get it working with aliases. To do that I modified the plugin to store connection-specific stuff not globally in the context, but in an alias-specific part of it. Would you be interested in merging that? And would you consider that a solution for this issue, or should I create a separate one?
